bench/lib.js

```diff
@@ -9,26 +9,56 @@ const ListDef = {
   Cons: ['x', 'xs'],
   Nil: []
 }
+const GiantDef = {
+  V0: [],
+  V1: ['a'],
+  V2: 'ab'.split(''),
+  V3: 'abc'.split(''),
+  V4: 'abcd'.split(''),
+  V5: 'abcde'.split(''),
+  V6: 'abcdef'.split(''),
+  V7: 'abcdefg'.split(''),
+  V8: 'abcdefgh'.split(''),
+  V9: 'abcdefghi'.split('')
```

Comment:
Let's use `['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i']` here. It's not too verbose.